Core/src/org/sleuthkit/autopsy/casemodule/Bundle.properties

```diff
@@ -136,6 +136,7 @@ Case.createCaseDir.exception.cantCreate=Cannot create case dir\: {0}
 Case.createCaseDir.exception.cantCreateCaseDir=Could not create case directory\: {0}
 Case.createCaseDir.exception.cantCreateModDir=Could not create modules output directory\: {0}
 Case.createCaseDir.exception.gen=Could not create case directory\: {0}
+Case.OpenEventChannel.ErrMsg=Case opening process failed to connect to nodes collaborating on case {0}.
```

Comment:
You speak the truth! I can hear myself saying the same thing about other abbreviations, like "evt" for "event." I should change it for the sake of consistency with self! I can get that in the next pull request for this feature.